enterprise/com/src/main/java/org/neo4j/com/storecopy/StoreCopyServer.java

```diff
@@ -175,28 +176,31 @@ public RequestContext flushStoresAndStreamStoreFiles( String triggerName, StoreW
                     File file = meta.file();
                     int recordSize = meta.recordSize();
 
-                    // Read from paged file if mapping exists. Otherwise read through file system.
-                    // A file is mapped if it is a store, and we have a running database, which will be the case for
-                    // both online backup, and when we are the master of an HA cluster.
-                    final Optional<PagedFile> optionalPagedFile = pageCache.getExistingMapping( file );
-                    if ( optionalPagedFile.isPresent() )
+                    boolean isDefaultFileSystem = pageCache.getCachedFileSystem() instanceof DefaultFileSystemAbstraction;
+                    if ( !isDefaultFileSystem )
                     {
-                        try ( PagedFile pagedFile = optionalPagedFile.get() )
+                        // Read from paged file if mapping exists. Otherwise read through file system.
+                        // A file is mapped if it is a store, and we have a running database, which will be the case for
+                        // both online backup, and when we are the master of an HA cluster.
+                        final Optional<PagedFile> optionalPagedFile = pageCache.getExistingMapping( file );
+                        if ( optionalPagedFile.isPresent() )
```

Comment:
now this presence condition is also not the same as before, if mapping not present then nothing will happen

Reply:
?
There is no else so it will go out to the try catch with ` fileSystem.open`.